### DISCUSSION TITLE
Quasar examples not working standalone but the components load in jupyter notebook example

### DISCUSSION THREAD

**Discussion by omdaniel** [Kitware/trame#406] [Q&A] [answered]
When I try to run the \examples\validation\panel\app_trame.py standalone I get this error in the terminal

 JS Error => Lookup error: Vue.use(Quasar, {})
The opengl vtk viewer loads but no quasar components

also \examples\validation\core\33_outside_lib.py

However, the quasar components load in the jupyter notebook just fine  \examples\validation\panel\trame.ipynb

also the panel example the scene_orientation_widget is not updating when navigating the scene, although maybe this a pyvista issue

I am running these on a Windows computer

**Comment by jourdain** [comment, marked as answer]
Did you skip the install step described in the readme?

`pip install trame trame-quasar trame-vtk`

Otherwise, try another port `--port 1234`

**Comment by jourdain** [comment]
If the other port was the solution, your browser cached a previous application that was vue2 while that one relies on vue3.

**Reply by omdaniel** [reply]
Setting the port to 1234 fixes all of the quasar examples, the app_panel works fine with port 8080 but the quasar example has issues with the default. Also the orientation widget still seems to be broken, it shows up but doesn't update

**Reply by omdaniel** [reply]
It works in the panel example, the widget in the quasar example seems to be broken

**Reply by jourdain** [reply]
What do you mean by `the widget in the quasar example seems to be broken`?

<img width="1194" alt="Screenshot 2024-01-05 at 13 44 07" src="https://github.com/Kitware/trame/assets/294518/d6ca2841-cd73-493d-9de3-4de7651156a1">
